.github/workflows/cron_nightly_build.yml

```diff
@@ -67,7 +67,14 @@ jobs:
         run: php tools/build/CreateRelease.php --destination-dir=${{ env.RELEASE_DIR }}
 
       - name: Rename build
-        run: today=`date +%Y-%m-%d-`; for i in *; do mv $i $today$GH_BRANCH-$i; done
+        run: today=`date +%Y-%m-%d`; for i in *; do mv $i $today-$GH_BRANCH-$i; done
+        working-directory: ${{ env.RELEASE_DIR }}
+
+      - name: Create nightly.zip for develop
+        if: matrix.BRANCH == 'develop'
+        run: |
+          today=`date +%Y-%m-%d`
+          cp $today-$GH_BRANCH*.zip nightly.zip
```

Comment:
So it means we can download the build from develop for each day right? And the one named `nightly.zip` always matches the last build from develop?

The concept is interesting but it means only develop has a nightly build, maybe it would be interesting to have the same principle for each branch? Like `develop-nighty.zip` `8.0.x-nighty.zip` `8.1.x-nighty.zip`

That's just a suggestion though, not blocking as it can always be added later

Reply:
> So it means we can download the build from develop for each day right? And the one named nightly.zip always matches the last build from develop?
Yes that's the point, to simplify automation scripts, one name "nightly.zip" always refer to the latest nightly.

Other branches can follow the path later on obviously ;)